### ISSUE TITLE
Specifying multiple targets in command line does not work as expected.

### ISSUE THREAD

**Issue by hubuk** [Microsoft/msbuild#1899]
**MSBuild version:**
```
Microsoft (R) Build Engine version 15.1.548.43366
```

**.NET Core Host version:**
```
Microsoft .NET Core Shared Framework Host
Version  : 1.1.0
Build    : 928f77c4bc3f49d892459992fb6e1d5542cb5e86
```

Following project is using `SuppressXunitContent` property to control whether the 'build' asset of `xunit.runner.visualstudio` package will be excluded from build or not. This asset copies some xunit dlls to output directory:
```XML
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net452</TargetFramework>
  </PropertyGroup>
  <!--
  By default xunit.runner.visualstudio package will copy test adapter
  to output directory via Build asset.
  Use: dotnet msbuild "/t:Restore;Rebuild" /p:SuppressXunitContent=False
  -->
  <ItemGroup Condition="'$(SuppressXunitContent)'=='False'">
    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
  </ItemGroup>
  <!--
  This will ignore Build asset from xunit.runner.visualstudio nuget package.
  As a result no xunit files will be copied to output directory.
  Use: dotnet msbuild "/t:Restore;Rebuild" /p:SuppressXunitContent=True
  -->
  <ItemGroup Condition="'$(SuppressXunitContent)'=='True'">
    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" ExcludeAssets="Build" />
  </ItemGroup>
</Project>
```
**Steps to reproduce:**
1. In a fresh project directory run `dotnet msbuild "/t:Restore;Rebuild" /p:SuppressXunitContent=False`.
**Expected behavior:** xunit dll files are copied to output directory.
**Actual behavior:** :exclamation:no dll files are copied.
2. `dotnet msbuild "/t:Rebuild" /p:SuppressXunitContent=False`.
**Expected behavior:** xunit dll files are copied to output directory.
**Actual behavior:** as expected.
3. Run `dotnet msbuild "/t:Restore;Rebuild" /p:SuppressXunitContent=True` in a project directory.
**Expected behavior:** xunit dll files were removed from output directory.
**Actual behavior:** :exclamation:dll files are still in the output directory.
4. `dotnet msbuild "/t:Rebuild" /p:SuppressXunitContent=True`.
**Expected behavior:** xunit dll files are removed from output directory.
**Actual behavior:** as expected.

*The problem occurs when both targets are specified in one command. Moving them to separate `dotnet msbuild` commands works as expected.*

5. Delete `out` and `bin` directories then run `dotnet msbuild "/t:Restore" /p:SuppressXunitContent=False` followed by `dotnet msbuild "/t:Rebuild" /p:SuppressXunitContent=False`
**Expected behavior:** xunit dll files are copied to output directory.
**Actual behavior:** as expected.
6. Run `dotnet msbuild "/t:Restore" /p:SuppressXunitContent=True` followed by `dotnet msbuild "/t:Rebuild" /p:SuppressXunitContent=True`
**Expected behavior:** xunit dll files are removed from output directory.
**Actual behavior:** as expected.

**Comment by rainersigwald** [comment]
If you're only seeing this with the `Rebuild` target, it's because of #1065, fixed by #1665 for a future update release. Separating the invocations is the best workaround for now.

**Comment by rainersigwald** [comment]
Also and totally unrelatedly: personally, I'd replace

```xml
  <!--
  By default xunit.runner.visualstudio package will copy test adapter
  to output directory via Build asset.
  Use: dotnet msbuild "/t:Restore;Rebuild" /p:SuppressXunitContent=False
  -->
  <ItemGroup Condition="'$(SuppressXunitContent)'=='False'">
    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
  </ItemGroup>
  <!--
  This will ignore Build asset from xunit.runner.visualstudio nuget package.
  As a result no xunit files will be copied to output directory.
  Use: dotnet msbuild "/t:Restore;Rebuild" /p:SuppressXunitContent=True
  -->
  <ItemGroup Condition="'$(SuppressXunitContent)'=='True'">
    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" ExcludeAssets="Build" />
  </ItemGroup>
```

with the equivalent-but-much-shorter

```xml
  <ItemGroup>
    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0">
      <!-- Avoid copying xunit files to the output directory when called with
           dotnet msbuild "/t:Restore;Rebuild" /p:SuppressXunitContent=True -->
      <ExcludeAssets Condition="'$(SuppressXunitContent)'=='True'">Build</ExcludeAssets>
    </PackageReference>
  </ItemGroup>
```

**Comment by hubuk** [comment]
@rainersigwald Thanks for the hint. Making `ExcludeAssets` an element simplifies a lot.
And regarding your previous comment:

> If you're only seeing this with the Rebuild target, it's because of #1065, fixed by #1665 for a future update release.

I have checked **Build** i.e. `Restore;Build`. It does not work either. Is this the same problem already addressed by #1665?
